tests/functional/object/rgw/test_obc_quota.py

```
@@ -85,7 +83,10 @@ def test_rgw_obc_quota(
                 ignore_error=False,
             )
         except CommandFailed as e:
-            logger.info(f"Quota {quota} blocked writes as expected: {e}")
+            if err_msg in str(e):
+                logger.info(f"Quota {quota} worked as expected!!")
```

Ruff (0.16.2)

[warning] Logging statement uses f-string

(G004)